src/routes/player.tsx

```diff
@@ -36,16 +35,13 @@ export default function Player() {
           className={`w-60 h-60 bg-alabaster-100 border-[1px] border-black shadow-md rounded-full self-center`}
           variants={itemVariants}
         ></motion.div>
-        <div className={`flex flex-row justify-around`}>
-          <div className={``}>
+        <div className={`flex flex-row justify-center items-center`}>
```

Comment:
We shouldn't use `mr-60`. Instead there should be a parent container that holds 2 containers that are spaced apart. The **green square** being the parent container. Thats what the `justify-around` was being used for previously. The **blue squares** being the child containers, which in the case of the 3 action buttons, can be used to control spacing with `spacing-x-{whatEverNumber} _instead_ of `mr-{number}`.
<img width="637" alt="Screenshot 2024-05-04 at 1 48 20 PM" src="https://github.com/dizzyjaguar/improvEar/assets/50177345/8cd86cae-db69-44c8-af04-b0a0bb1293d1">

Reply:
**Also**,
In terms of commit name, there is no need to include the word `Tailwind`. A better commit message would be _Update record and play button styling_ 😄

Reply:
So, I made the changes per your comment above. I am coming across an issue with the spacing-x-number. Even though I placed that in the child component, it seems that nothing is happening on the localhost server. Example: 
<div className={`spacing-x-10`}>
           <RecordButton />
          </div>
          
 Record button on localserver shows no changes.

Reply:
@mouakiang  can you post a fuller snippet of this? Of the div and the child comps

Reply:
![Code Snippet](https://github.com/dizzyjaguar/improvEar/assets/116847348/a2e95fa2-d25f-48ec-9743-70116520de40)

Reply:
![snippet](https://github.com/dizzyjaguar/improvEar/assets/116847348/242a7498-b4a0-4166-906d-5aff86019e8a)